DigitalLearningSolutions.Data.Tests/DataServices/CourseDataServiceTests.cs

```diff
@@ -473,6 +473,32 @@ public void GetCoursesAvailableToCentreByCategory_returns_active_and_inactive_al
             result.Should().ContainEquivalentOf(expectedInactiveCourse);
         }
 
+        [Test]
+        public void GetApplicationsAvailableToCentreByCategory_returns_expected_values()
+        {
+            // Given
+            const int centreId = 101;
+            int? categoryId = null;
+
+            // When
+            var result = courseDataService.GetApplicationsAvailableToCentreByCategory(centreId, categoryId).ToList();
+
+            // Then
+            var expectedFirstApplication = new ApplicationDetails
+            {
+                ApplicationId = 1,
+                ApplicationName = "Entry Level - Win XP, Office 2003/07 OLD",
+                PLAssess = false,
+                DiagAssess = false,
+                CourseTopicId = 3,
+                CategoryName = "Office 2007",
+                CourseTopic = "Microsoft Office",
+            };
+
+            result.Should().HaveCount(423);

```

Comment:
We've have a pattern of using `AssertionScope` whenever there are multiple assertions to make.
There are places we haven't done this but it's good to use `AssertionScope` so that logs can contain all failures for that test rather than just the detail of the first failed assertion.

```
using(new AssertionScope())
{
  // your assertions
}
```

Reply:
Thanks for pointing that out, made the change. I'll make sure to use AssertionScope going forward